images/runtime/php/8.1/base.bullseye.Dockerfile

```diff
@@ -1,11 +1,9 @@
-# DisableDockerDetector "Below image not yet supported in the Docker Hub mirror"
-FROM php:8.1-bullseye
+FROM oryxdevmcr.azurecr.io/private/oryx/php-8.1
 SHELL ["/bin/bash", "-c"]
 ENV PHP_VERSION 8.1.6
 
 RUN a2enmod rewrite expires include deflate remoteip headers
-#Bake in client certificate path into image to avoid downloading it
-ENV PATH_CA_CERTIFICATE="/etc/ssl/certs/ca-certificate.crt"
```

Comment:
Should we keep this line?

Reply:
yeah I think I'm going to revert this file to its previous state